src/pages/career.tsx

```diff
@@ -6,16 +6,31 @@ import { CareerPage } from '../components/pages/career/career-page';
 import { SyPersonioJob } from '../types';
 import { IGatsbyImageData } from 'gatsby-plugin-image';
 
+export interface OfficeImage {
```

Comment:
This type already exists in the landingpage. I would either import it from there or maybe create a `src/types/types.ts` and import it from there in both landingpage and career.

Reply:
Thanks for the suggestion. I have finally import it from `src/pages/index`